.github/workflows/trunk.yml

```diff
@@ -120,6 +120,26 @@ jobs:
       docker-image: ${{ needs.linux-bionic-py3_7-clang9-slow-build.outputs.docker-image }}
       test-matrix: ${{ needs.linux-bionic-py3_7-clang9-slow-build.outputs.test-matrix }}
 
+  linux-focal-py3_7-clang7-tsan-build:
+    name: linux-focal-py3.7-clang7-tsan
+    uses: ./.github/workflows/_linux-build.yml
+    with:
+      build-environment: linux-focal-py3.7-clang7-tsan
+      docker-image-name: pytorch-linux-focal-py3-clang7-asan
```

Comment:
do you need to register this new docker-image in the ECR registry?

Reply:
Oh, this is not a new docker image, I re-use the existing docker image for ASAN.  It would be better to call this docker something like `pytorch-linux-focal-py3-clang7-sanitizers` instead but it's minor